test/hotspot/jtreg/testlibrary/ctw/src/sun/hotspot/tools/ctw/CtwRunner.java

```diff
@@ -52,6 +53,14 @@ public class CtwRunner {
     private static final Predicate<String> IS_CLASS_LINE = Pattern.compile(
             "^\\[\\d+\\]\\s*\\S+\\s*$").asPredicate();
 
+    /**
+     * Value of {@code -Dsun.hotspot.tools.ctwrunner.ctw_extra_args}. Extra
+     * comma-separated arguments to pass to CTW subprocesses.
+     */
+    public static final String CTW_EXTRA_ARGS
```

Comment:
how about you use getProperty("sun.hotspot.tools.ctwrunner.ctw_extra_args", "").
by giving it an empty string as default value, you can take iout if (null != CTW_EXTRA_ARGS) below. 

btw, you may also need to update the year in the copyrights header.

Reply:
Yup. Made these changes.

Reply:
Better to make CTW_EXTRA_ARGS private. If it's needed outside this class, add a public accessor. Don't think it's needed outside though.

Reply:
Made it private. Not needed outside.